cms/static/js/factories/manage_users.js

```diff
@@ -1,203 +1,31 @@
-                                                    }
-                                                }
-                                            }
-                                        });
-                                        prompt.show();
-                                    }
-                                });
-                            }
-                        },
-                        secondary: {
-                            text: gettext('Cancel'),
-                            click: function(view) {
-                                view.hide();
-                            }
-                        }
-                    }
-            });
-            msg.show();
-        });
-
-        $('.toggle-admin-role').click(function(event) {
-            event.preventDefault();
-            var type, url, role;
-            if($(this).hasClass('add-admin-role')) {
-                role = 'instructor';
-            } else {
-                role = 'staff';
-            }
-
-            url = $(this).closest('li[data-url]').data('url');
-            $.ajax({
-                url: url,
-                type: 'POST',
-                dataType: 'json',
-                contentType: 'application/json',
-                notifyOnError: false,
-                data: JSON.stringify({role: role}),
-                success: function(data) {location.reload();},
-                error: function(jqXHR, textStatus, errorThrown) {
-                    var message, prompt;
-                    try {
-                        message = JSON.parse(jqXHR.responseText).error || unknownErrorMessage;
-                    } catch (e) {
-                        message = unknownErrorMessage;
-                    }
-                    prompt = new PromptView.Error({
-                        title: gettext("There was an error changing the user's role"),
-                        message: message,
-                        actions: {
-                            primary: {
-                                text: gettext('Try Again'),
-                                click: function(view) {
-                                    view.hide();
-                                }
-                            }
-                        }
-                    });
-                    prompt.show();
-                }
-            });
-        });
+    return function (containerName, users, tplUserURL, current_user_id, allow_actions) {
+        function updateMessages(messages) {
+            var local_messages = _.extend({}, messages);
+            local_messages.alreadyMember.title = gettext('Already a course team member');
+            local_messages.deleteUser.messageTpl = gettext(
+                'Are you sure you want to delete {email} from the course team for “{container}”?'
+            );
+            return local_messages;
+        }
+
+        var options = {
+            el: $("#content"),
```

Comment:
@cahrens this view binds "Add New User" event, and events only bound to elements under `el`. So it needs to wrap both actual content and navbar. Unfortunately, `#content` is a closest common parent for navbar and content.